### DISCUSSION TITLE
EBSI

### DISCUSSION THREAD

**Discussion by Ducmor** [e-id-admin/open-source-community#11] [General]
Mich würde interessieren wie sehr sich die Schweiz auf eine eigene Lösung fokussiert und was die Abhängigkeiten zu EBSI sind.
https://ec.europa.eu/digital-building-blocks/wikis/display/EBSI/Home

Mit Hyperledger sind wir heute ja auch schon auf dem gleichen "Standard" wie die EU?

**Comment by christianheimann** [comment]
Es ist derzeit natürlich noch vieles in Bewegung, auch in der EU – und es wird noch lange in Bewegung bleiben. Bis im Oktober 2022 sollten in der EU mit der angekündigten "Toolbox" aber bereits die wichtigsten Eckpfeiler eingeschlagen worden sein. Die damit einhergehende Standardisierung kommt dem Schweizer Vorhaben sicher entgegen.

Das Projektteam wie auch das Team rund um die Pilotprojekte verfolgen die Entwicklungen, auch finden punktuelle Gespräche mit Personen aus dem Kreis EBSI/EU und weiteren europäischen Ländern statt.

Der Fokus liegt also derzeit darauf, vom Bestehenden "das Beste" zu übernehmen/nutzen; den gesamten Techstack neu erfinden zu wollen ist weder zielführend noch realistisch.

**Reply by Ducmor** [reply]
Danke Christian für das Update.

**Comment by pascalgottret** [comment]
So wie ich das mitbekommen habe, läuft bei der EU ein Projekt, das die bestehende Lösung EBSI 1.0 basierend auf Hyperledger ablösen soll. Hyperledger mangelt es an Skalierbarkeit, wobei nun Protokolle gesucht werden die dieses Problem lösen. Zudem darf der Faktor Sicherheit und Kosten nicht ausser Acht gelassen werden. Wir wollen nicht für jedes neue DID Document oder eine Aktualisierung dieses Gebühren zahlen.

Ich bin gespannte auf die weiteren Entwicklungen bei EBSI. Es werden viele spannende Protokolle geprüft.
